index.js

```diff
@@ -1,7 +1,56 @@
 // Iteration 1: Names and Input
+console.log("I`m ready");
 
+let hacker1 = "The driver's name is Leal";
+let hacker2 = "The navigator's name is Duarte";
 
 // Iteration 2: Conditionals
 
+let driverName = hacker1.slice(21);
+let navigatorName = hacker2.slice(24);
+
+
+
+let numOfCharDriver = driverName.length;
+let numOfCharNavigator = navigatorName.length;
+
+
+if( driverName.length > navigatorName.length){
+    console.log("The driver has the longest name, it has " + numOfCharDriver + " characters");
+} 
+if(driverName.length < navigatorName.length){
+    console.log("It seems that the navigator has the longest name, it has "+ numOfCharNavigator + " characters.");
+}
+else {
+    console.log("Wow, you both have equally long names, " + numOfCharDriver + " characters!");
+}
+
 
 // Iteration 3: Loops
+
+//3.1
+for (let i = 0; i < driverName.length; i++) {
```

Comment:
Here, the idea was to print all the characters of the driver's name, separated by a space and in capitals i.e. "L E A L".

A good solution would be:

```
let withSpace = '';
for (let i = 0; i < hacker1.length; i++) {
     withSpace += hacker1.[i] + ' ';
 }
console.log(withSpace.toUpperCase());
```